### DISCUSSION TITLE
Impossible to connect to my Tickets listing

### DISCUSSION THREAD

**Discussion by GreggBazin22** [ovh/manager#12561] [Q&A]
Hi there,

recently I had to order 270 domain names. And I associated MXPLAN for each of these domain names.

And I think it was a red flag for OVH security robots  :-)  I had to change credit card 3 times!

But now, I can't access to my support tickets. And I can't create a new one. Everytime I'm trying to connect, I'm redirecting to this URL:  https://ca.ovh.com/auth/?onsuccess=https%3A%2F%2Fca.ovh.com%2Fauth%2Fsso%2Fsaml%2F%3FSAMLRequest

I don't understand why I have "ca" instead of "www" in the URL. It asks me to login again. And I have the message "Error: Invalid Account ID or password"

I'm in Indonesia but even with a VPN based in Europe I have the same issue. And it was not the case before this huge order. I live in Indonesia since one year and first time I can't access like that.

Do you have any idea? Thanks in advance for your time and your help!

Gregg

**Comment by entropicalabs-systems** [comment]
This is happening to us as well, and we can't find a way to get in touch with support about it.

@GreggBazin22 Did you ever figure it out?

**Comment by GreggBazin22** [comment]
Hi there, sometimes it happens again. I manage with my VPN but it's not always working. Very weird

**Reply by entropicalabs-systems** [reply]
I once got in by logging in in incognito mode (but the login flow was still "strange"). Funny thing is... there is no other way to get in touch with someone to have them have a look at it. I'll check if it's any different if I connect from different regions with a VPN, as you suggested.
